packages/docregistry/src/context.ts

```diff
@@ -219,6 +223,14 @@ export class Context<
     return this._readyPromise;
   }
 
+  /**
+   * Returns whether this document context is writable. True only if the
+   * contents model is writable and collaboration is disabled.
+   */
+  get writable(): boolean {
```

Comment:
Ok then can we make it `protected` to not increase the surface API (as it is used only locally) and could we use another name to avoid misunderstanding with the `this._contentsModel.writable` attribute that is closer to the purpose; something like:

```suggestion
   * Whether the document support saving through the Contents REST API or not.
   */
  protected get isSavingWithContents(): boolean {
```

> I'm not great at wording so feel free to propose something else.